hooks/tk-multi-publish2/basic/nuke_update_flame_clip.py

```diff
@@ -211,10 +275,7 @@ def publish(self, settings, item):
         """
 
         # update shot clip xml file with this publish
-        try:
-            self._update_flame_clip(item)
-        except Exception, e:
-            raise("Unable to update Flame clip xml: %s" % (e,))
+        self._update_flame_clip(item)
```

Comment:
why was this removed? Will the removal lead to obscure/hard to read error messages in case the xml is malformed?

Reply:
I should add that back. The downside to what was there before is that it smothers the stack, so it's tough to debug the code when something is raised during the XML update. However, the error message is cleaner with the way it was before. I'll set it back up the way it was...I only changed it while debugging.